### PR TITLE
add checked Table

### PR DESCRIPTION
**Pull request by zhouzilong2020**

### CODE REVIEW

**Comment by zhouzilong2020** [comment]
PTAL

File: frontend/src/components/CostTableChecked.vue [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <heroicons-outline:chevron-up class="ml-1 h-6" v-show="!isExpended" />
+      </div>
+    </n-button>
+
+    <cost-table
+      v-show="isExpended"
+      class="my-2"
+      virtual-scroll
+      :data-row="dataRow"
+      :isLoading="isLoading"
```
Comment (LiuJi-Jim) [inline frontend/src/components/CostTableChecked.vue:16]: use kebab-case when binding properties. e.g. `:is-loading`.

**Review by LiuJi-Jim — COMMENTED** [review]
LGTM overall, with some minor comments

File: frontend/src/components/CostTableChecked.vue (hunk 1 above)
Comment (zhouzilong2020) [reply frontend/src/components/CostTableChecked.vue:16]: This should be a bug in GitHub. I have changed it to is-loading.

**Review by LiuJi-Jim — APPROVED** [review]
LGTM

**Pull request merged**